target/linux/qualcommax/ipq60xx/base-files/etc/hotplug.d/firmware/11-ath11-caldata

```diff
@@ -18,6 +18,10 @@ case "$FIRMWARE" in
 	netgear,wax214)
 		caldata_extract "0:art" 0x1000 0x10000
 		;;
+	tplink,eap610-outdoor)
+		# /tmp/factory_data should be mounted by preinit
+		cp /tmp/factory_data/radio "/lib/firmware/$FIRMWARE"
```

Comment:
Check how EAP660 does this, and then you can avoid having to fixup WLAN MAC-s later by simply patching the caldata instead

Reply:
That does not make sense to me architecturally. I find it far more readable to set the MAC directly rather than have to patch an undocumented binary format. How do we guarantee that newer wifi firmwares will be able to parse the "patched" caldata?

Apart from the technical uncertainties, I prefer to keep the current approach because I find it much more logical and readable.